### DISCUSSION TITLE
Welcome to AtomAI Discussions!

### DISCUSSION THREAD

**Discussion by ziatdinovmax** [pycroscopy/atomai#87] [General]
Feel free to share your thoughts, ask questions, and make (constructive) suggestions!

**Comment by Pragalbhv** [comment]
Dear Ziatdinov,
Thank you so much for the library; I particularly find the segmentation utility immensely helpful.
I am facing problems in running the Im2spec notebook, though; the Dropbox dataset link is not working. Could you please take a look?

Thank you!

**Comment by ziatdinovmax** [comment]
@Pragalbhv - thanks for using the package and apologies for this inconvenience. This dropbox was linked to my ORNL account and since I'm not working there anymore, everything got deleted.
For the STEM EELS, please use this google drive link instead:
```!gdown -O "eels_imspec.npz" https://drive.google.com/uc?id=1BprChho_y-LdVQstR1vYzCxmK677algR```
It doesn't have ```specval``` but you need them only for plotting, not for actual analysis.
I will try to find SPM BEPS one as well.

**Reply by Pragalbhv** [reply]
Thank you so much @ziatdinovmax !
